BUILD

```diff
@@ -72,6 +72,9 @@ genrule(
     touch $@
     """,
     stamp = True,
+    # This action writes to a non-hermetic output location, so running it
+    # remotely isn't currently possible.
```

Comment:
this isn't about running it remotely, just running it at all since it's write files outside of bazel's execroot

Reply:
Sure, I just copied the comment, but you're right of course - I'll update it.